src/js/element.js

```diff
@@ -0,0 +1,133 @@
+import { ELEMENT_COLORS } from "./element_colors";
+import { PERIODIC_TABLE } from "./periodic_table.js";
+import { CONVERSION, UNITS, convertLength, convertEnergy } from "./units";
+
+export class Element {
```

Comment:
Let's use `ChemicaElement` pls, for clarity. In JS there are many other Elements - HTML, https://element.eleme.io/#/en-US, etc.

Reply:
Class name has been renamed